scripts/build-color-variants.js

```diff
@@ -480,11 +480,13 @@ function buildColorVariants(variables, config) {
       const colorValue = variables[color];
       return result += stripIndent(`
         .shadow-${color}-on-hover:hover,
-        .shadow-${color}-on-active.is-active {
+        .shadow-${color}-on-active.is-active,
+        .shadow-${color}-on-active.is-active:hover {
```

Comment:
Hmm looks like they need to be decoupled from [being grouped together](https://github.com/mapbox/assembly/blob/573999d2bbdfd4f5ce9d4d2d95b9199cb96dfcd4/test/__snapshots__/build-color-variants.jest.js.snap#L9-L12) which means more lines.